subsys/bluetooth/controller/ll_sw/openisa/hal/RV32M1/radio/radio.c

```diff
@@ -167,7 +167,7 @@ static void pkt_rx(void)
 #define IRQ_MASK ~(GENFSK_IRQ_CTRL_T2_IRQ_MASK | \
 		   GENFSK_IRQ_CTRL_RX_WATERMARK_IRQ_MASK | \
 		   GENFSK_IRQ_CTRL_TX_IRQ_MASK)
-void isr_radio(void)
+void isr_radio(void *arg)
 {
 	u32_t tmr = GENFSK->EVENT_TMR & GENFSK_EVENT_TMR_EVENT_TMR_MASK;
```

Comment:
Please add, ARG_UNUSED(arg);

Reply:
Adeed

Reply:
If not in a hurry to merge PR, move this line up to be the 1st in the function